Gemfile

```diff
@@ -95,6 +95,9 @@ gem 'rack-protection', git: 'https://github.com/finnlabs/rack-protection.git', r
 # https://github.com/kickstarter/rack-attack
 gem 'rack-attack'
 
+# Patch Rails HTML whitelisting for Angular curly braces
+gem 'rails-angular-xss', github: 'opf/rails-angular-xss'
```

Comment:
I seem to remember you saying that this can be removed once we have Angular 1.5 in place. If that is so, I'd like a commit in here so that we remove this gem once we no longer need it.